container_images/daisy-builder/main.sh

```diff
@@ -37,8 +51,16 @@ if [[ "$JOB_TYPE" == "presubmit" ]]; then
   DAISY_VARS+=",pull_ref=pull/${PULL_NUMBER}/head"
 fi
 
+## generate buildID
+if [[ "$JOB_TYPE" == "postsubmit" ]]; then
+  BUILD_ID=$(GENERATE_NEW_VERSION)
+  DAISY_VARS+=",build_id=${BUILD_ID}"
```

Comment:
can we change all references from 'build_id' to 'version' ? build id is something different

Reply:
done